src/target/metadata_module.cc

```diff
@@ -119,6 +78,52 @@ static runtime::metadata::Metadata ConvertMetaData(
   return runtime::metadata::Metadata(std::move(n));
 }
 
+static runtime::Module CreateCrtMetadataModule(
```

Comment:
could you move this up where it was above ConvertMetaData so we can see if there are any diffs here?